x-pack/metricbeat/module/aws/ec2/_meta/data.json

```diff
@@ -0,0 +1,26 @@
+{
+    "@timestamp":"2016-05-23T08:05:34.853Z",
+    "beat":{
+        "hostname":"beathost",
+        "name":"beathost"
+    },
+    "metricset":{
+        "host":"localhost",
+        "module":"aws",
+        "name":"ec2",
+        "rtt":44269
+    },
+    "aws":{
```

Comment:
Based on the current fields it's not possible to tell from which ec2 instance the data comes from. Can we enrich it with data like the `arn` or the hostname? What info do we have about each instance we collect metrics for?

Reply:
Currently I only have instance_id here to identify which ec2 instance the data belongs to. We can definitely add more info here in addition to InstanceId. For example, InstanceType, OwnerId, IP, tags and etc.

Reply:
I see you get a lot of info from the [DescribeInstances](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeInstances.html) call, which is performed already.

I would say most of these fields are useful, I would copy them to your working document to map them to field names. Some of them can map to existing fields, which is really interesting for future correlation (see [host metadata](https://www.elastic.co/guide/en/beats/filebeat/master/add-host-metadata.html) and its [ECS definition](https://github.com/elastic/ecs#host) as an example).

I suggest you start by adding a few (`host` metadata is a good start) in the code, and review the full list later.